src/index.ts

```diff
@@ -330,7 +330,8 @@ export default runExtension({
       onloadArgs
     );
     const toggleSortReferences = runSortReferences();
-
+    if (!!extensionAPI.settings.get("sort-references")) {toggleSortReferences(true);}
```

Comment:
That's probably it!

As an fyi, I'd add the [Prettier](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode) extension to your VSCode workspace. It will auto format the code to look "prettier" on save, based on what we have configured